crml/sylo/src/groups/mod.rs

```diff
@@ -398,6 +401,32 @@ decl_module! {
 
 			Ok(())
 		}
+
+		#[weight = SimpleDispatchInfo::FixedOperational(0)]
+		fn migrate_groups(origin, group_id: T::Hash, group: Group<T::AccountId, T::Hash>) -> DispatchResult {
+			ensure_signed(origin.clone())?;
```

Comment:
We don't need to `ensure_signed` as ensure_sylo_migrator already checks it

Reply:
Done